models/topics.models.js

```diff
@@ -12,3 +12,18 @@ exports.fetchTopics = () => {
       return rows;
     });
 };
+
+exports.fetchTopicsBySlug = (topic) => {
+  if (!topic) {
+    return;
+  }
+  return db
+    .query(
+      `SELECT * FROM topics
+    WHERE slug = $1`,
+      [topic]
+    )
+    .then(({ rows }) => {
+      return rows.length > 0 ? true : false;
+    });
```

Comment:
This is an interesting function for 2 reasons!
- It's a database query that's only returning a boolean?
- I don't think it's actually being invoked anywhere thanks to your solution in the `articles.model`? I did check your `topics.controller` and that only references the `fetchTopics` above.

Does this need to be removed? Thankfully, if you remove it you should relatively easily be able to run all the tests and give yourself confidence it's not a problem (or find out what breaks). The beauty of testing, eh!

Reply:
Ah, I can take this out!

It's a vestigial function from when I was trying to do some (what I thought was very clever, but just failed) ternary stuff in my topics.controller to decide whether or not to pass through the topic argument to fetchArticles :joy: